gammapy/background/models.py

```diff
@@ -71,3 +81,825 @@ def evaluate(self, x, y):
+        axes : `~matplotlib.pyplot.axes`
+            axes of the figure, optional
+        image : !!!!!!!!!!!!
+            !!!!!!!!!!!!!!!!, optional(??!!)
+        """
+        import matplotlib.pyplot as plt
+        from matplotlib.colors import LogNorm
+        from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+        do_only_1_plot = False # general case: print all plots
+        if energy:
+            energy = energy.flatten() # flatten
+            # check shape of energy: only 1 value is accepted
+            nvalues = len(energy)
+            if nvalues != 1:
+                raise IndexError("Expected exactly 1 value for energy, got {}."
+                                 .format(nvalues))
+            else:
+                energy = Quantity(energy[0])
+                print("Reqested plot only for 1 energy: {}".format(energy))
+                do_only_1_plot = True
+
+        n_energy_bins = len(self.energy_bins) - 1
+        nimages = n_energy_bins
+        ncols = 4
+        nrows = 4
+        if do_only_1_plot:
+            n_energy_bins = nimages = ncols = nrows = 1
+        npads_per_canvas = ncols*nrows
+
+        fig, axes = plt.subplots(nrows=nrows, ncols=ncols)
+        fig.set_size_inches(35., 35., forward=True)
+        count_images = 1
+        count_canvases = 1
+        count_pads = 1
+
+        extent = self.image_extent
+        energy_bin_centers = self.spectrum_bin_centers
+        if do_only_1_plot:
+            # find energy bin containing the specified energy
+            energy_bin, energy_bin_edges = self.find_energy_bin(energy)
+            ss_energy_bin_edges = "[{0}, {1}) {2}".format(energy_bin_edges[0].value,
+                                                          energy_bin_edges[1].value,
+                                                          energy_bin_edges.unit)
+            print("Found energy {0} in bin {1} with boundaries {2}.".format(energy,
+                                                                            energy_bin,
+                                                                            ss_energy_bin_edges))
+
+        for ii in range(n_energy_bins):
+            if do_only_1_plot:
+                ii = energy_bin
+            print("ii", ii)
+            data = self.background[ii]
+            energy_bin_center = energy_bin_centers[ii]
+            print ("  image({}) canvas({}) pad({})".format(count_images, count_canvases, count_pads))
+
+            if do_only_1_plot:
+                fig.set_size_inches(8., 8., forward=True)
+                ax = axes
+            else:
+                ax = axes.flat[count_pads - 1]
+            image = ax.imshow(data.value,
+                              extent=extent.value,
+                              interpolation='nearest',
+                              norm=LogNorm(), # color log scale
+                              cmap='afmhot')
+            # TODO: image seems mirrored in Y axis w.r.t. image in fits!!!
```

Comment:
Probably you just have to call `imshow` with `origin=lower` ... this is a common thing:
https://github.com/astropy/photutils/search?utf8=%E2%9C%93&q=imshow

But getting axes orientations right in data and in plots can be tricky.
Maybe we can just postpone this to when you implement `fill` and `evaluate` methods ... then it should be easy to implement a test and see if if there is a y-axis orientation issue with the data array and / or the plot?